projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -41,6 +41,7 @@ public static function init() {
 
 		add_action( 'plugins_loaded', array( __CLASS__, 'load_first_posts_stream_helpers' ) );
 		add_action( 'plugins_loaded', array( __CLASS__, 'load_domain_email_nag' ) );
+		add_action( 'plugins_loaded', array( __CLASS__, 'load_block_patterns' ) );
```

Comment:
This function does not exist, which is what's causing tests on D129217-code to fail.